x-pack/platform/plugins/shared/alerting/server/application/rule/transforms/transform_raw_artifacts_to_domain_artifacts.test.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectReference } from '@kbn/core/server';
+import type { RawRule } from '../../../types';
+import { transformRawArtifactsToDomainArtifacts } from './transform_raw_artifacts_to_domain_artifacts';
+
+describe('transformRawArtifactsToDomainArtifacts', () => {
+  it('should return empty artifacts if rawArtifacts is undefined', () => {
```

Comment:
```suggestion
  it('should return default artifacts if rawArtifacts is undefined', () => {
```

The word empty confused me.

Reply:
See 6e5ca07c4fa77890c3774267fcb1b87518524f23